packages/adapter-vercel/index.js

```diff
@@ -13,14 +13,14 @@ export default function (options) {
 		name: '@sveltejs/adapter-vercel',
 
 		async adapt({ utils }) {
-			const dir = '.vercel_build_output';
+			const dir = '.output';
 			utils.rimraf(dir);
 
 			const files = fileURLToPath(new URL('./files', import.meta.url));
 
 			const dirs = {
 				static: join(dir, 'static'),
-				lambda: join(dir, 'functions/node/render')
+				pages: join(dir, 'server/pages')
```

Comment:
surely `join(dir, 'server', 'pages')` would be preferable here?

Reply:
it's all the same to node, so i personally prefer `server/pages` as it's more compact and readable